operator/cmd/mesh/manifest-apply.go

```diff
@@ -133,9 +139,13 @@ func InstallCmd(logOpts *log.Options) *cobra.Command {
 
 func runApplyCmd(cmd *cobra.Command, rootArgs *rootArgs, maArgs *manifestApplyArgs, logOpts *log.Options) error {
 	l := clog.NewConsoleLogger(cmd.OutOrStdout(), cmd.ErrOrStderr(), installerScope)
+
+	obv := version.OperatorBinaryVersion.String()
+	detectIstioVersionDiff(obv, maArgs)
 	// Warn users if they use `manifest apply` without any config args.
 	if len(maArgs.inFilenames) == 0 && len(maArgs.set) == 0 && !rootArgs.dryRun && !maArgs.skipConfirmation {
-		if !confirm("This will install the default Istio profile into the cluster. Proceed? (y/N)", cmd.OutOrStdout()) {
+		prompt := fmt.Sprintf("This will install the default profile of Istio %s into the cluster. Proceed? (y/N)", obv)
```

Comment:
not from your PR, but this is clearly wrong "the default profile"? It is hardcoded. I would just remove that and say "This will install Istio 1.x into the cluster"

Reply:
Simple and easy to understand. Thanks for your suggestion.